<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>AddRepository</class>
  <widget class="QDialog" name="AddRepository">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>466</width>
        <height>274</height>
      </rect>
    </property>
    <property name="modal">
      <bool>true</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
        <number>0</number>
      </property>
      <item row="2" column="0">
        <widget class="QLabel" name="errorText">
          <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="minimumSize">
            <size>
              <width>0</width>
              <height>20</height>
            </size>
          </property>
          <property name="font">
            <font>
              <pointsize>11</pointsize>
            </font>
          </property>
          <property name="text">
            <string />
          </property>
          <property name="textFormat">
            <enum>Qt::PlainText</enum>
          </property>
          <property name="scaledContents">
            <bool>false</bool>
          </property>
          <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item row="1" column="0">
        <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="currentIndex">
            <number>0</number>
          </property>
          <widget class="QWidget" name="tabWidgetPage1">
            <attribute name="title">
              <string>General</string>
            </attribute>
            <layout class="QFormLayout" name="repoDataFormLayout">
              <property name="fieldGrowthPolicy">
                <enum>QFormLayout::ExpandingFieldsGrow</enum>
              </property>
              <property name="leftMargin">
                <number>5</number>
              </property>
              <property name="topMargin">
                <number>5</number>
              </property>
              <property name="rightMargin">
                <number>5</number>
              </property>
              <property name="bottomMargin">
                <number>5</number>
              </property>
              <item row="0" column="1">
                <widget class="QLabel" name="title">
                  <property name="font">
                    <font>
                      <bold>true</bold>
                    </font>
                  </property>
                  <property name="text">
                    <string>Initialize New Backup Repository</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="0">
                <widget class="QLabel" name="repoLabel">
                  <property name="text">
                    <string>Repository URL:</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="leftMargin">
                    <number>0</number>
                  </property>
                  <property name="topMargin">
                    <number>0</number>
                  </property>
                  <item>
                    <widget class="QLineEdit" name="repoURL">
                      <property name="placeholderText">
                        <string>ssh://abc123@abc123.repo.borgbase.com/./repo</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QToolButton" name="chooseLocalFolderButton">
                      <property name="toolTip">
                        <string>Choose a local folder</string>
                      </property>
                      <property name="text">
                        <string>...</string>
                      </property>
                      <property name="icon">
                        <iconset>
                          <normaloff>:/icons/folder-open.svg</normaloff>:/icons/folder-open.svg</iconset>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QToolButton" name="useRemoteRepoButton">
                      <property name="toolTip">
                        <string>Choose a remote repository</string>
                      </property>
                      <property name="text">
                        <string>...</string>
                      </property>
                      <property name="icon">
                        <iconset>
                          <normaloff>:/icons/globe.svg</normaloff>:/icons/globe.svg</iconset>
                      </property>
                    </widget>
                  </item>
                </layout>
              </item>
              <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                  <property name="text">
                    <string>Borg passphrase:</string>
                  </property>
                </widget>
              </item>
              <item row="2" column="1">
                <widget class="QLineEdit" name="passwordLineEdit">
                  <property name="enabled">
                    <bool>true</bool>
                  </property>
                  <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                  </property>
                </widget>
              </item>
              <item row="3" column="1">
                <widget class="QLineEdit" name="confirmLineEdit">
                  <property name="enabled">
                    <bool>true</bool>
                  </property>
                  <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                  </property>
                </widget>
              </item>
              <item row="3" column="0">
                <widget class="QLabel" name="confirmLabel">
                  <property name="text">
                    <string>Confirm passphrase:</string>
                  </property>
                </widget>
              </item>
              <item row="4" column="0" colspan="2">
                <widget class="QLabel" name="passwordLabel">
                  <property name="text">
                    <string>TextLabel</string>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
          <widget class="QWidget" name="tabWidgetPage2">
            <attribute name="title">
              <string>Advanced</string>
            </attribute>
            <layout class="QFormLayout" name="advancedFormLayout">
              <property name="fieldGrowthPolicy">
                <enum>QFormLayout::ExpandingFieldsGrow</enum>
              </property>
              <property name="leftMargin">
                <number>5</number>
              </property>
              <property name="topMargin">
                <number>5</number>
              </property>
              <property name="rightMargin">
                <number>5</number>
              </property>
              <property name="bottomMargin">
                <number>5</number>
              </property>
              <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                  <property name="text">
                    <string>SSH Key:</string>
                  </property>
                </widget>
              </item>
              <item row="0" column="1">
                <widget class="QComboBox" name="sshComboBox">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <item>
                    <property name="text">
                      <string>Automatically choose SSH Key (default)</string>
                    </property>
                  </item>
                </widget>
              </item>
              <item row="1" column="0">
                <widget class="QLabel" name="encryptionLabel">
                  <property name="text">
                    <string>Encryption:</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="1">
                <widget class="QComboBox" name="encryptionComboBox">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                </widget>
              </item>
              <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                  <property name="text">
                    <string>Extra Borg Arguments:</string>
                  </property>
                </widget>
              </item>
              <item row="2" column="1">
                <widget class="QLineEdit" name="extraBorgArgumentsLineEdit" />
              </item>
            </layout>
          </widget>
        </widget>
      </item>
      <item row="3" column="0">
        <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <resources />
  <connections />
</ui>